cmd/components.go

```diff
@@ -17,7 +17,7 @@ import (
 
 var ComponentsCmd = &cobra.Command{
 	Use:   "components",
-	Short: "List all Dapr components",
+	Short: "Lists all Dapr components",
```

Comment:
We should keep "list" verses "lists".  List is the used word when the noun is plural
az aks list
e.g.     list               **: List managed Kubernetes clusters.**
Docker 
  images      **List images**

Reply:
"list" (command) yes. But to keep consistency with the other descriptions, "lists" (in the 3rd person) is how we explain the subject ("list" command).

https://www.merriam-webster.com/dictionary/list

Reply:
Fixed. If we want to change back to 3rd person, we can talk. They are all in model verbs.